sympy/polys/polytools.py

```diff
@@ -6218,8 +6218,20 @@ def sqf_list(f, *gens, **args):
     (2, [(x + 1, 2), (x + 2, 3)])
 
     """
-    return _generic_factor_list(f, gens, args, method='sqf')
-
+    options.allowed_flags(args, ['frac', 'polys'])
+    opt = options.build_options(gens, args)
+    f = sympify(f)
```

Comment:
`_sympify`

Reply:
`_sympify` isn't defined in the file

Reply:
`from sympy.core.sympify import _sympify`

Reply:
Thanks. I wanted to confirm that functions with "_" prefix are for calling within i.e they are not accessible outside, is that the case?